### PR TITLE
[Label] Changed orientation(string) to angle(number)

### LINKED ISSUE #2048: Poor and unclear documentation of label orientation
Can you guys make it easier to understand the direction in which labels will render? 

This has no description for "orientation":
http://plottablejs.org/docs/classes/plottable.component.axislabel.html#constructor

...and this is unclear: ("The orientation of the Label (horizontal/left/right) (default = "horizontal")")
http://plottablejs.org/docs/classes/plottable.component.label.html#constructor

Really, we're talking about the rotation of the text:
- 0 deg (default, horizontal left-to-right), 90 deg (vertical top-to-bottom), and -90 deg (vertical bottom-to-top)

Ideally, these would be (self-documenting...) string constants with some description... somewhere. :)

### PR DESCRIPTION
**Pull request by rcchen**
We no longer use orientation strings, and instead pass the angle straight in. 

* `constructor(displayText = "", orientation = "horizontal")` -> `constructor(displayText = "")`
* `orientation(orientation: string)` -> `angle(angle: number)`

Fixes #2048

### CODE REVIEW

File: test/components/labelTests.ts [hunk 1]
```diff
@@ -26,7 +26,8 @@ describe("Labels", () => {
   // Skipping due to FF odd client bounding rect computation - #1470.
   it.skip("Left-rotated text is handled properly", () => {
```
Comment (jtlan) [inline test/components/labelTests.ts]: Can we un-skip this?

File: test/components/labelTests.ts [hunk 2]
```diff
@@ -40,7 +41,8 @@ describe("Labels", () => {
   // Skipping due to FF odd client bounding rect computation - #1470.
   it.skip("Right-rotated text is handled properly", () => {
```
Comment (jtlan) [inline test/components/labelTests.ts]: and un-skip this too?

File: src/components/label.ts [hunk 3]
```diff
@@ -81,28 +80,26 @@ export module Components {
...
+        return this._angle;
       } else {
-        orientation = orientation.toLowerCase();
-        if (orientation === "horizontal" || orientation === "left" || orientation === "right") {
-          this._orientation = orientation;
+        if (angle === -90 || angle === 0 || angle === 90) {
+          this._angle = angle;
         } else {
-          throw new Error(orientation + " is not a valid orientation for LabelComponent");
+          throw new Error(angle + " is not a valid angle for LabelComponent");
```
Comment (jtlan) [inline src/components/label.ts]: " is not a valid angle for Label"

**Comment by jtlan** [comment]
[-1] Un-skip the tests?

**Comment by rcchen** [comment]
@aicioara let me know when your test un-skipping stuff is done, then this one can go in.

**Comment by aicioara** [comment]
I can get the label stuff in already

**Pull request merged**